cypress/cypress/helpers/getSelectedText.ts

```diff
@@ -0,0 +1,9 @@
+const getSelectedTextFromInputField = (input: JQuery<HTMLElement>): string => {
```

Comment:
I think a nicer way of doing this would be to have a cypress custom command:

https://docs.cypress.io/api/cypress-api/custom-commands

But I leave it to you to change it or not :-)

Reply:
i wasn't aware of this custom command's, nice and light solution, hopefully as you expected :)